pom.xml

```diff
@@ -1,362 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
@TikhomirovSergey Does it create any issues if we keep this file just as it is as ```pom.xml``` for sometime?